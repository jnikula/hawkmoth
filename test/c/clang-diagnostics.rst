
.. c:function:: int frob(struct list *list, enum mode mode)

   struct list and enum mode aren't declared.

