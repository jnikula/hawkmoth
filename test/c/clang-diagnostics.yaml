domain: c
directive: autodoc
directive-arguments:
  - clang-diagnostics.c
errors: clang-diagnostics.stderr
expected: clang-diagnostics.rst
