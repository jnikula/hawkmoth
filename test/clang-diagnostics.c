/**
 * struct list and enum mode aren't declared.
 */
int frob(struct list *list, enum mode mode);
