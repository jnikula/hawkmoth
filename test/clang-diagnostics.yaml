domain: c
directive: autodoc
directive-arguments:
  - clang-diagnostics.c
errors: clang-diagnostics.stderr
