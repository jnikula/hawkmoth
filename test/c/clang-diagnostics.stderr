WARNING: clang-diagnostics.c:4: declaration of 'struct list' will not be visible outside of this function
WARNING: clang-diagnostics.c:4: declaration of 'enum mode' will not be visible outside of this function
